fuzz/slh-dsa.c

```diff
@@ -318,7 +318,8 @@ static void slh_dsa_sign_verify(uint8_t **buf, size_t *len, void *key1,
     unsigned char *sig = NULL;
     OSSL_PARAM params[4];
     int paramidx = 0;
-    int intval;
+    int intval1, intval2;
```

Comment:
Nit: perhaps calling these with the names of the actual params would be better. But approving anyway.